src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1630,16 +1630,6 @@ public override BoundNode VisitUsingStatement(BoundUsingStatement node)
 
                     default:
                         Debug.Assert(local.DeclarationKind == LocalDeclarationKind.UsingVariable);
-                        int slot = GetOrCreateSlot(local);
-                        if (slot >= 0)
-                        {
-                            SetSlotAssigned(slot);
-                            NoteWrite(local, value: null, read: true);
```

Comment:
It is done when we visit the variable declaration inside the using.

---
In reply to: [79029103](https://github.com/dotnet/roslyn/pull/13807#discussion_r79029103) [](ancestors = 79029103)